spec/fluent.ebnf

```diff
@@ -69,8 +73,8 @@ block-expression     ::= select-expression
                        | variant-list
 
 select-expression    ::= inline-expression __ '->' __ variant-list
-attribute-expression ::= identifier '.' identifier
-variant-expression   ::= identifier '[' _? variant-key _? ']'
+attribute-expression ::= message-identifier '.' identifier
```

Comment:
If this is just `identifier`, the syntax defines that we can't dereference attributes of private messages?

That'd make inline-expression safe, and we could use an unsafe-inline-expression that adds private-indentifier.identifier as selector-expression instead of inline-expression?

Reply:
Nice, I like this approach. It makes syntax work for us without adding more... syntax.

We could even go further and define `selector-expression` which would be different from `inline-expression` and `block-expression` and could comprise of:

```
inline-expression    ::= quoted-text
                       | number
                       | identifier
                       | external
                       | variant-expression
                       | call-expression
                       | placeable

block-expression     ::= select-expression
                       | variant-list

selector-expression  ::= quoted-text
                       | number
                       | identifier
                       | external
                       | attribute-expression
                       | call-expression
```

Compare it with:

https://github.com/projectfluent/fluent/blob/b9552ae754c373311e79594a73a90a29b4540ef5/spec/fluent.ebnf#L60

Reply:
Yeah, agreed, allowing placeable or variant-expression doesn't make much sense in a selector-expression